src/data.rs

```diff
@@ -22,19 +20,22 @@ pub(crate) struct NodeData {
     pub(crate) other_layout_cache: Option<Cache>,
     /// Does this node's layout need to be recomputed?
     pub(crate) is_dirty: bool,
+
+    /// wether or not the Node has custom measure data attached to it
```

Comment:
I'd like a bit more information here about what exactly "measure data" is.

Reply:
Totally, I haven't put much effort into docs yet. But will try to elaborate